modules/multiplayer/doc_classes/MultiplayerSynchronizer.xml

```diff
@@ -53,7 +53,7 @@
 	</methods>
 	<members>
 		<member name="delta_interval" type="float" setter="set_delta_interval" getter="get_delta_interval" default="0.0">
-			Time interval between delta synchronizations. When set to [code]0.0[/code] (the default), delta synchronizations happen every network process frame.
+			Time interval between delta synchronizations. Used when the replication is set to [code]On Change[/code]. A value of [code]0.0[/code] (the default) means that delta synchronizations happen every network process frame.
```

Comment:
I know it looks more daunting, but if you want to be accurate you could refer to the constant itself, which will redirect the user to a more proper explanation.
The existing "when set to" reads better in my opinion, maybe it should be "if" for simplicity.
```suggestion
			Time interval between delta synchronizations. Used when the replication is set to [constant SceneReplicationConfig.REPLICATION_MODE_ON_CHANGE]. If set to [code]0.0[/code] (the default), delta synchronizations happen every network process frame.
```
Alternatively, but bear in mind this is much worse for localization:
```suggestion
			Time interval between delta synchronizations. Used when the replication is set to [b]On Change[/b]. If set to [code]0.0[/code] (the default), delta synchronizations happen every network process frame.
```